BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/distance/BeaconDistanceCalculator.java

```diff
@@ -23,6 +23,23 @@ public static float calculateDistanceTo(Beacon beacon) {
         return calculateDistanceTo(beacon, beacon.getFilteredRssi());
     }
 
+    /**
+     * Use this method to remove the altitude from the distance to the beacon. Calculation based on
+     * Pythagoras with altitude to calculate more accurate distance to the beacon, if the distance
+     * is double the altitude.
+     */
+    public static float calculateDistanceWithoutAltitudeDelta(Beacon beacon, float rssi) {
+        double altitude = beacon.getLocation().getAltitude();
+        float distance = calculateDistanceTo(beacon, rssi);
+        // distance should be double of the altitude to make pythagoras meaningful
+        if (altitude > 0 && distance > (altitude * 2)) {
```

Comment:
Using `altitude` like this will create different values for different heights at which to phone is (different positions, different heights of users). For small rooms this may create greater errors as to using the distance without `altitude`.
To counter this we should use a average phone height instead of using incorrect `altitudes` for the `beacons`.

Reply:
Will be addressed in the next pr